backend/news_api_scraper.py

```
@@ -0,0 +1,495 @@
+            if opp.published_at:
+                # Parse ISO date
+                publish_date = datetime.fromisoformat(opp.published_at.replace('Z', '+00:00'))
+                hours_old = (datetime.now() - publish_date.replace(tzinfo=None)).total_seconds() / 3600
+                
+                if hours_old <= 24:
+                    return 80  # Very recent news
+                elif hours_old <= 72:
+                    return 60  # Recent news
+                else:
+                    return 40  # Older news
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)